src/main/java/net/sf/jabref/gui/groups/TransferableEntrySelection.java

```diff
@@ -11,7 +11,7 @@
 
 import net.sf.jabref.model.entry.BibEntry;
 
-class TransferableEntrySelection implements Transferable {
+public class TransferableEntrySelection implements Transferable {
```

Comment:
This is a heritage from when I tried to implement drag and drop. currently it is not needed.